### ISSUE TITLE
Handling multiple frequencies with the same trip_id

### ISSUE THREAD

**Issue by cbergy** [catalogueglobal/datatools-ui#249]
Importing a new feed does not take into account multiple frequencies that use the same trip_id: only the first one is shown in the editor. 

The same happens when manually editing frequencies: the editor blocks you from inserting the same trip_id for different frequencies.

The GTFS standard does not pose this limitations.
See the example on the [Google site](https://developers.google.com/transit/gtfs/examples/gtfs-feed#frequenciestxt)

**Comment by landonreed** [comment]
Yes, this limitation should have been documented.  We were aware of the limitation when manually editing frequencies, which for the moment we deemed tolerable because it was still producing valid GTFS.  However, on import it should not lose track of the multiple frequency entries per trip_id.
